From 4cca790be6a69f829ca7e67e1a70c1b4b0f80a01 Mon Sep 17 00:00:00 2001
From: Meng Mingming <mingming.meng@nxp.com>
Date: Wed, 27 Dec 2017 10:17:52 +0800
Subject: [PATCH 3174/5242] MLK-17311-3 drm,imx: Add struct
 drm_imx_dpu_frame_info

commit  4827d19ea5f58b11b84817e1bd2578db90c5a6f0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add struct drm_imx_dpu_frame_info.

Signed-off-by: Meng Mingming <mingming.meng@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/drm/imx_drm.h |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/include/uapi/drm/imx_drm.h b/include/uapi/drm/imx_drm.h
index b285388..e80cad4b 100644
--- a/include/uapi/drm/imx_drm.h
+++ b/include/uapi/drm/imx_drm.h
@@ -24,6 +24,22 @@
 extern "C" {
 #endif
 
+/**
+ * Dpu frame info.
+ *
+ */
+struct drm_imx_dpu_frame_info {
+	__u32   width;
+	__u32   height;
+	__u32	x_offset;
+	__u32	y_offset;
+	__u32	stride;
+	__u32	format;
+	__u64   modifier;
+	__u64	baddr;
+	__u64	uv_addr;
+};
+
 #define DRM_IMX_DPU_SET_CMDLIST                 0x00
 #define DRM_IMX_DPU_WAIT                        0x01
 #define DRM_IMX_DPU_GET_PARAM                   0x02
-- 
1.7.9.5

